archaius2-core/src/main/java/com/netflix/archaius/property/DefaultPropertyContainer.java

```diff
@@ -174,35 +174,6 @@ public void removeListener(PropertyListener<T> listener) {
         }
         
         /**
-         * Update to the latest value and return either the new value or previous value if not updated
-         * or failed to resolve.
-         * 
-         * @param latestVersion
-         * @return
-         */
-        T updateAndGet(int latestVersion) {
-            int[] cacheVersion = new int[1];
-            T existing = cache.get(cacheVersion);
-            try {
-                T value = resolveCurrent();
-                if (value != existing && (value == null || existing == null || !existing.equals(value))) {
```

Comment:
There were a few other use cases but essentially yes.  If the value was anything but new the version timestamp never got updated.